README.Rmd

```diff
@@ -31,14 +33,69 @@ The **analysis** directory contains:
 
 ## Additional files required for analysis
 
-For surface annotation of differentially expressed genes, surface protein annotations were pulled from the Cell Surface Protein Atlas website {https://wlab.ethz.ch/cspa/#abstract}. 
+For surface annotation of differentially expressed genes, surface protein annotations were pulled from the [Cell Surface Protein Atlas website](https://wlab.ethz.ch/cspa/#abstract). 
 
 > Bausch-Fluck D, Hofmann A, Bock T, Frei AP, Cerciello F, et al. (2015) *A Mass Spectrometric-Derived Cell Surface Protein Atlas*. PLoS One <https://doi.org/10.1371/journal.pone.0121314>
 
-### Licenses
+## Licenses
 
 **Code :** MIT License, Copyright (c) 2020 Roosheel Patel
 
 The organization of this repository was adapted from the rrtools research compendium structure
 
-> Marwick, B. (2017). Computational reproducibility in archaeological research: Basic principles and a case study of their implementation.*Journal of Archaeological Method and Theory*, 24(2), 424-450.<https://doi.org/10.1007/s10816-015-9272-9>
\ No newline at end of file
+> Marwick, B. (2017). Computational reproducibility in archaeological research: Basic principles and a case study of their implementation.*Journal of Archaeological Method and Theory*, 24(2), 424-450.<https://doi.org/10.1007/s10816-015-9272-9>
+
+## Session Information
+
+The following session information (compute environment and R/package versions) will reproduce the analysis and figures generated in the manuscript.
```

Comment:
Reword:
Session information (computer environment and R/package versions) were used to generate the analyses and figures in the manuscript:

Reply:
Fixed according to comments